icd/generated/function_declarations.h

```diff
@@ -340,7 +340,7 @@ static const std::unordered_map<std::string, uint32_t> device_extension_map = {
     {"VK_EXT_pageable_device_local_memory", 1},
     {"VK_KHR_maintenance4", 2},
     {"VK_ARM_shader_core_properties", 1},
-    {"VK_KHR_shader_subgroup_rotate", 1},
+    {"VK_KHR_shader_subgroup_rotate", 2},
```

Comment:
good catch!

Reply:
wait, im confusing, this was from https://github.com/KhronosGroup/Vulkan-Tools/commit/3af5fdabdd9fed697b01f18b1fad0ee00f86268e

why is not showing the diff from `main`

Reply:
This PR pre-dates the update to the 1.3.277 header. This just needs rebasing.

Reply:
I wish GitHub was capable of showing the dif against the targeted branch when the targeted branch changed.... regardless, it probably would have been fine to merge, but didn't want to take a chance with it causing some strange git artifact on Github